components/card-section.html

```diff
@@ -15,7 +15,37 @@
 <body>
```

Comment:
Give this body a class of `card-section` for better referencing in you `scss`

```
.card-section {
    // Then you can nest properly. This will `give .card-section .blank`
    .blank {
    }
}